i18n/en.json

```diff
@@ -2412,6 +2412,18 @@
     "translation": "[{{ .SiteName }}] Your password has been updated"
   },
   {
+    "id": "api.templates.user_access_token_body.info",
+    "translation": "A user access token was added to your account on {{ .SiteURL }}. They can be used by integrations to access {{.SiteName}} with your account.<br>If this change wasn't initiated by you, please contact your system administrator."
```

Comment:
Yeah, I think it's important to be clear in this case that anyone who has access to this token can access your account